src/to-govspeak.js

```diff
@@ -88,6 +88,21 @@ service.addRule('removeEmptyParagraphs', {
   replacement: () => ''
 })
 
+// remove style elements
+service.addRule('style', {
+  filter: ['style'],
+  replacement: () => ''
+})
+
+// strip paragraph elements within list items
+service.addRule('stripParagraphsInListItems', {
```

Comment:
This name is pretty clear, but just to double-check: markdown does support this, but it looks like we're making a compromise and choosing not to convert it? Perhaps we could explain in the commit :-).

Reply:
Without this rule we'd end up with an extra line in between list items.  I'll add this to the commit message to explain it.